docs/api-reference/event-manager.md

```diff
@@ -39,6 +39,7 @@ Creates a new `EventManager` instance.
 *  `options.events` {Object} -  Map of {event name: handler} to register on init.
 *  `options.recognizers` - {Object}  Gesture recognizers from Hammer.js to register, as an Array in [Hammer.Recognizer format](http://hammerjs.github.io/api/#hammermanager)
 *  `options.rightButton` - {Boolean}  Recognizes click and drag from pressing the right mouse button. Default `false`. If turned on, the context menu will be disabled.
+*  `options.touchAction` - {String}  Allow browser default touch actions. Default `none`. See [hammer.js doc](http://hammerjs.github.io/touch-action/).
```

Comment:
Would be good to link to https://github.com/uber/react-map-gl/issues/506 here to hint at what this should be set to for users having issues with page scroll / map pan interference.